internal/api/middleware/timer.go

```diff
@@ -0,0 +1,39 @@
+package middleware
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/go-kit/kit/metrics"
+)
+
+func getIdentifier(ctx context.Context) string {
+	str := strings.ReplaceAll(chi.RouteContext(ctx).RoutePattern(), "/", ".")
+	regx := regexp.MustCompile(`[{}]`)
+
+	str = regx.ReplaceAllString(str, "")
+	str = strings.Replace(str, ".", "", 1)
```

Comment:
These last two lines can be written `return str[1:]`, taking all but the first (index 0) element of the slice.